src/stactools/sentinel1/rtc_metadata.py

```diff
@@ -147,11 +149,14 @@ def epsg(self) -> Optional[str]:
     def metadata_dict(self):
         ''' match s2 l2a cogs from https://earth-search.aws.element84.com/v0 '''
         sentinel_metadata = {
+            # NOTE: once PySTAC adds MGRS extension can move this to stac.py
+            'mgrs:utm_zone': self.metadata['TILE_ID'][:2],
+            'mgrs:latitude_band': self.metadata['TILE_ID'][2],
+            'mgrs:grid_square': self.metadata['TILE_ID'][3:],
             'sentinel:mgrs': self.metadata['TILE_ID'],
-            'sentinel:utm_zone': self.metadata['TILE_ID'][:2],
-            'sentinel:latitude_band': self.metadata['TILE_ID'][2],
-            'sentinel:grid_square': self.metadata['TILE_ID'][3:],
             'sentinel:product_ids': self.grd_ids,
+            # NOTE: change to 'valid_percent' in raster extension?
+            # https://github.com/stac-extensions/raster#statistics-object
```

Comment:
Seems like it'd be worth using the `raster` extension's attribute, since it's more widely used than `sentinel`? What's the downside of using `raster`?